### DISCUSSION TITLE
Override the background of in-line assistant

### DISCUSSION THREAD

**Discussion by boetro** [zed-industries/zed#18075] [Feature Requests]
Hi! I was wondering how I can override the background color of the inline assistant suggestions. For example the big blue block in the below screen shots:

<img width="556" alt="Screenshot 2024-09-19 at 9 58 03 AM" src="https://github.com/user-attachments/assets/ff4f26a9-368a-42c6-a615-38cd0198c8d9">

**Comment by tomgenoni** [comment]
Add this to your `settings.json` and experiment with the colors. Unfortunately, the `deleted.background` value doesn't update instantly, you have to invoke the inline assistant each time.

```
// Inline assistant color overrides
"experimental.theme_overrides": {
  "deleted.background": "#2C313Cff",
  "info.background": "#1E2227ff",
  "info.border": "#3D4452ff"
}
```

![Screenshot 2024-10-08 at 7 42 02 PM](https://github.com/user-attachments/assets/b64c5cbd-931c-47eb-a14b-7d7fe5af2da7)

I'm using the above overrides with the "One Dark Pro" theme. Without these overrides I was getting a bright red background for the existing code and that bright blue background for the suggested code, making both unreadable. 

Here's an example [full theme](https://raw.githubusercontent.com/zed-industries/zed/refs/heads/main/assets/themes/one/one.json) with (I think) all possible values.

**Reply by boetro** [reply]
Perfect, thanks!
